packages/runtime-core/src/component.ts

```diff
@@ -870,12 +869,16 @@ function setupStatefulComponent(
         setupContext,
       ],
     )
+    const isAsyncSetup = isPromise(setupResult)
```

Comment:
Tweaked the lines here so all the `const` declarations are together for better minification